Program.cs

```diff
@@ -0,0 +1,31 @@
+using System.Configuration;
```

Comment:
if system. configuration is not being used remove it.

Reply:
Done